src/point.js

```diff
@@ -23,9 +23,9 @@ SVG.Point = SVG.invent({
       return new SVG.Point(this)
     }
     // Morph one point into another
-  , morph: function(point) {
+  , morph: function(x, y) {
       // store new destination
-      this.destination = new SVG.Point(point)
+      this.destination = new SVG.Point(x, y)
```

Comment:
Well this change is just to allow _morph_ to be passed the coordinates as parameters. With this change, all of the following are valid ways to call _morph_: 

```javascript
pt.morph({x:3, y:4}) 
pt.morph([3, 4])
pt.morph(3, 4)
```
There is no need to make any changes to the constructor since it can already be passed two parameters. In fact, I made this change mostly to make the method _morph_ more consistent with the constructor.